e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -646,7 +648,18 @@ describe("scenarios > embedding > dashboard parameters with defaults", () => {
 });
 
 describeEE("scenarios > embedding > dashboard appearance", () => {
+  const originalBaseUrl = Cypress.config("baseUrl");
   beforeEach(() => {
+    // This is needed to "restore" the initial value after having navigated to an html file like
+    // `cy.visit(`e2e/test/scenarios/embedding/embedding-dashboard.html?iframeUrl=${iframeUrl}`)`
+    // Doing that apparently sets the baseUrl to null and cypress will fail the next cy.request() call
+    // with the following error:
+    // > CypressError: `cy.request()` must be provided a fully qualified
+    // > `url` - one that begins with `http`. By default `cy.request()` will use
+    // > either the current window's origin or the `baseUrl` in
+    // > `e2e/support/cypress.config.js`. Neither of those values were present.
+    Cypress.config("baseUrl", originalBaseUrl);
```

Comment:
For anyone who is still confused about this line. Because of the way [`Cypress.config`](https://docs.cypress.io/api/cypress-api/config#Test-Configuration-vs-Cypressconfig) works, it only is reset when the whole suite is completed, so this is a way to restore the value back in between tests.

By the way, it's used here.
https://github.com/metabase/metabase/blob/40c4aef99ce8fdba68d49a7e17c42859834eff50/e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js#L942